src/nv/storage/public.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
     Error, Result, WrapperErrorKind,
 };
 use log::error;
-use std::convert::{TryFrom, TryInto};
+use std::convert::{From, TryFrom, TryInto};
```

Comment:
I don't think you need to import `From`, it gets included in the [prelude](https://doc.rust-lang.org/std/prelude/index.html)

Reply:
Fixed